tuna/machine.py

```diff
@@ -161,23 +180,24 @@ def connect(self, abort=None):
 
     return connection
 
-  def get_num_cpus(self):
+  def get_num_cpus(self) -> int:
     """return number of available cpus"""
+    stdout: TextIO
     _, stdout, _ = self.connect().exec_command('nproc')
     self.num_cpus = int(stdout.readline())
 
     return self.num_cpus
 
-  def get_avail_gpus(self):
+  def get_avail_gpus(self) -> List[Dict[int, str]]:
     """return list of available gpus"""
     if not self.avail_gpus:
       if not self.gpus:
         self.get_properties()
-        self.avail_gpus = range(len(self.gpus))
+        self.avail_gpus = range(len(self.gpus))  #type: ignore
```

Comment:
don't ignore type.

Reply:
error: Incompatible types in assignment (expression has type "range", variable has type "List[int]") 
used list() o enable the same